app/src/main/java/io/gnosis/safe/di/components/ApplicationComponent.kt

```diff
@@ -35,14 +36,18 @@ interface ApplicationComponent {
 
     fun viewModelFactory(): ViewModelProvider.Factory
 
-    fun repositories(): Repositories
-
     fun appDispatchers(): AppDispatchers
 
     fun termsChecker(): TermsChecker
 
     fun tracker(): Tracker
 
+    fun safeRepository(): SafeRepository
```

Comment:
Are these required here? I would assume that this is not required as they are only used via view models that are create via the `ViewModelFactory` ... if this is not the case I would guess that something is not set up correctly

Reply:
I think it was left over from a previous approach. Thanks for noticing.